targets/TARGET_Freescale/TARGET_K20XX/TARGET_TEENSY3_1/device/system_MK20DX256.c

```diff
@@ -8,6 +8,8 @@
 **     
 **
 **     Version:             rev. 1.0, 2011-12-15
+                            rev. 1.1, 2018-02-26 (Aidan Walton)
```

Comment:
Just wondering why this 120MHz support for K20DX is not there by default? Is this system file out of date (2011, is there newer with this clock config) ?

Reply:
The max CPU frequency for the xxx7 family is 72MHz. As Aidan mentioned, 96MHz and 120MHz is using the part outside the specified range. There is no guarantee this will reliably function.